modules/ipi-install-configuring-the-install-config-file.adoc

```diff
@@ -15,6 +15,17 @@ The installation program no longer needs the `clusterOSImage` {op-system} image
 
 . Configure `install-config.yaml`. Change the appropriate variables to match the environment, including `pullSecret` and `sshKey`.
 +
+[NOTE]
+====
+Set the number of replicas to `0` in the `compute` stanza to deploy a three-node cluster. This cluster consisting of three control plane machines provides a smaller, more resource efficient cluster that can be used for testing, development, and production. 
```

Comment:
Is it worth to mention that the we have only the options worker, replicas: 0, 2+ and we don't have the option to install cluster with only 1 worker?